upgrades/steps_28.go

```diff
@@ -117,15 +118,16 @@ func moveUnitAgentStateToController(ctx Context) error {
 		return nil
 	}
 
-	var fileNames []string
+	// For now a CAAS cannot be machineTagKind so it will not come as far as here for k8.
```

Comment:
CAAS will never have machine tags

Reply:
removed this check and added more detail in commit msg